crud.py

```diff
@@ -0,0 +1,53 @@
+from sqlalchemy import select
+from sqlalchemy.orm import Session
+
+import schemas
+from db import models
+
+
+def get_all_authors_paginated(db: Session, page: int = 0, size: int = 10):
+    offset = page * size
+    selected_authors = select(models.Author).offset(offset).limit(size)
+    return db.execute(selected_authors).scalars().all()
+
+
+def get_all_books_paginated(
+        db: Session, author_id: int, page: int = 0, size: int = 10
+):
+    queryset = db.query(models.Book)
```

Comment:
Consider using `select` instead of `query` for consistency with other parts of the code. This will also align with modern SQLAlchemy practices.